app/views/posts/show.html.erb

```diff
@@ -1,4 +1,35 @@
-<h1>
-This shows the details of apost with id = #{post.find(:id)}
-written by an author = #{users.find(:id)}
-</h1>
+<div class="posts-wrapper">
+  <h1 class="header-text"> Here is list of posts for an author with author_id = id </h1>
+
+  <div class="posts-container">
+    <div class="author-info user-link-container">
+      <img class="post-box" src = "/box-post.png" alt="Post logo" >
+      <div class="user-link-wrapper">
+      <p class="post-link"> <%= link_to @user.name, user_path(@user) %> </p>
+          <p class="posts-counter"> Number of posts: <%= @user.posts_counter %> </p>
+      </div>
+    </div>
+
+    <ul class="posts-list">
+      <% @user.posts.each do |post| %>
+        <li class="post-list-item">
+          <p class="post-title"> <%= link_to post.title, user_post_path(@user, post) %> </p>
+          <p class="post-text"> <%= post.text %> </p>
+          <p class="comments-likes-count">
+            Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
+          </p>
+        </li>
+
+       <div class="comments-container">
+        <% post.five_recent_comments.each do |comment| %>
+          <li class="comments-list">
+            <p class="comment-text"> <%= comment.author.name%>: <%= comment.text %> </p>
+          </li>
+        <% end %>
+       </div>
+      <% end %>
+    </ul>
```

Comment:
- [ ]  Right now, your post controller actions are empty. So it doesn't return any variable:

Here's a suggestion to write in your posts controller.
```
def index
@user = User.find(params[:user_id])
@posts= @user.posts
end
def show
@user = User.find(params[:user_id]
@post = @user.posts.find(params[:id])
end
```

-  [ ] I couldn't see any code in the post index template. This code that you put on the show template actually belongs to the index page which should show all the posts for a user. The post show page should looks like this and it should render all of the comments for a posts:
## The post show page
![image](https://github.com/obibaadoma/Blog_App/assets/91486791/ddfd692e-2594-4289-9bec-d658b028cbce)

## The posts index page
![image](https://github.com/obibaadoma/Blog_App/assets/91486791/a7d36675-9350-4de9-88d9-f211c14c8845)
- [ ] Please align the elements with flexbox to match exactly this, you should also add borders to elements so that we can check the exact spacing that you give to an element: 
![image](https://github.com/obibaadoma/Blog_App/assets/91486791/5d8fd2a6-2779-41da-921c-0a05c703d641)

Reply:
controllers rectified.